docs/dev/connectors/kafka.md

```diff
@@ -263,7 +255,7 @@ Example:
 {% highlight java %}
 final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
-FlinkKafkaConsumer09<String> myConsumer = new FlinkKafkaConsumer09<>(...);
```

Comment:
`docs/dev/event_time.md`, `docs/dev/event_time.zh.md`, `docs/dev/event_timestamps_watermarks.md` and `docs/dev/event_timestamps_watermarks.zh.md` contains a reference to `FlinkKafkaConsumer09`